e2e/action-row.spec.ts

```diff
@@ -0,0 +1,101 @@
+import { test, expect, type Page, type Locator } from '@playwright/test';
+
+const challengeButtons = [
+  'Instructions',
+  'index.html',
+  'styles.css',
+  'Console'
+];
+
+const editorButtons = ['index.html', 'styles.css'];
+
+let page: Page;
```

Comment:
```suggestion
import { test, expect, type Locator } from '@playwright/test';

const challengeButtons = [
  'Instructions',
  'index.html',
  'styles.css',
  'Console'
];

const editorButtons = ['index.html', 'styles.css'];
```

We also don't need a global `page` because...